static/library/bar_component.ts

```diff
@@ -53,6 +53,12 @@ export class DatacommonsBarComponent extends LitElement {
     ${unsafeCSS(tilesCssString)}
   `;
 
+  /**
+   * Draw as a stacked chart instead of grouped chart
+   */
+  @property({ type: Boolean })
+  stacked?: boolean;
```

Comment:
Nit: Let's add an example of a call using stacked=true to the example usage comment doc

Reply:
Done!